test/WebSites/Basic/Controllers/GenericsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Basic.Controllers
+{
+    /// <summary>
+    /// Summary for GenericsController
+    /// </summary>
+    [Route("/generics")]
+    [Produces("application/json")]
+    public class GenericsController
+    {
+        [HttpPost(Name = "CreateString")]
+        public string CreateString([FromBody] GenericType<string> genericString)
+        {
+            throw new NotImplementedException();
+        }
+
+        [HttpPost(Name = "CreateDateTime")]
+        public string CreateDateTime([FromBody] GenericType<DateTime> genericObject)
+        {
+            throw new NotImplementedException();
+        }
+    }
+}
```

Comment:
Not sure if this is the best place to put this

But it at least needs to be integration tested

Reply:
As long as a test would fail if this functionality got broken, it should be fine to live here. New projects are typically added when specific configurations are needed for the generation or for different types of apps.